src/components/navbar/navbar.js

```diff
@@ -103,6 +103,9 @@ class Navbar extends React.Component {
                 <a className="dropdown-item" onClick={ this.props.doLogout }>Logout</a>
               </div>
             </li>
+            <li className="nav-item">
+              <a className="nav-link help" href="https://docs.google.com/presentation/d/e/2PACX-1vTRSMVBEvObOl1sCKPmXMChP8A4eZScVmrRrzS6mDw0Imi5LkbFd1sSgqDS-QEPcBD-gvBFwmanrPIC/pub?start=true&loop=false&delayms=3000"alt="Help Documentation" target="_blank" rel="noopener noreferrer">Help</a>
```

Comment:
Is this a permalink? or will it change/go obsolete when the document is updated?  I suggest testing this or resolving it before we add this link.

Reply:
I believe the document automatically updates; the Google slide deck is under continual "publish" mode. We can wait, though.

Reply:
can confirm, the link does not change even when document edit (often use it for school). But link may change when duplicated, or when sharing status is changed.